ui/log-view/logviewer.css

```diff
@@ -0,0 +1,64 @@
+#root {
+  height: 100%;
+}
+
+.dropdown-menu {
+    top: inherit;
+}
+
+.result-status {
+    color: lightgrey;
```

Comment:
There's a `.lightgray` class in the `treeherder-global.css` file you could use instead.

Reply:
Good call.  Thanks.  :)